README.md

```diff
@@ -1 +1,72 @@
 # Portfolio
+
+> My second Portfolio project built with HTML, CSS and JavaScript. If you need anytjing like this, feel free to contact me at [Email](ragawilliam570@gmail.com). 
+
+## Built with
+
+- HTML
+- CSS
+
+## Setup for Basic HTML/CSS Project
+
+- Linter for HTML/CSS/Js installed
+
+## Getting Started
+
+- ### Mobile Version
+
+![Screenchot for mobile](./Templates/Template%203-main-mobile.svg)
+
+- ### Desktop Version
+
+![Screenshot for desktop](./Templates/Template%203-main.svg)
```

Comment:
Please only add the screenshots of your work don't try to copy the design image and pretend as its yours